### PR TITLE
Added create topic logic and a log

### PR DESCRIPTION
**Pull request by jeongkyun-oh**
## Proposed changes

- The topic is created automatically with Kafka broker configuration, but try topic creation when initializing Kafka structure and leaves a log if an error is returned.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: datasync/chaindatafetcher/kafka/kafka.go [hunk 1]
```diff
@@ -60,11 +60,23 @@ func NewKafka(conf *KafkaConfig) (*Kafka, error) {
...
+	}
+
+	err = kafka.CreateTopic(conf.getTopicName(EventBlockGroup))
+	if err != nil {
+		logger.Warn("creating a block group is failed", "err", err)
+	}
+	err = kafka.CreateTopic(conf.getTopicName(EventTraceBroup))
+	if err != nil {
+		logger.Warn("creating a trace group is failed", "err", err)
+	}
```
Comment (aidan-kwon) [inline datasync/chaindatafetcher/kafka/kafka.go:76]: Does this service work when the topics are not created?

Comment (jeongkyun-oh) [reply datasync/chaindatafetcher/kafka/kafka.go:76]: Yes, there is a broker option to create a topic automatically.

Comment (aidan-kwon) [reply datasync/chaindatafetcher/kafka/kafka.go:76]: Is there any reason you put these lines even though it is not necessary?

Comment (jeongkyun-oh) [reply datasync/chaindatafetcher/kafka/kafka.go:76]: Ah, it is necessary if a topic should be created with the given parameters. So, the topic wasn't created properly with the chaindatafetcher options. The topic that I used was used previously made. I leave the logs as well.

**Review by ehnuje — APPROVED** [review]

File: datasync/chaindatafetcher/kafka/kafka.go [hunk 2]
```diff
@@ -60,11 +60,23 @@ func NewKafka(conf *KafkaConfig) (*Kafka, error) {
...
+	kafka := &Kafka{
 		config:   conf,
 		producer: producer,
 		admin:    admin,
-	}, nil
+	}
+
+	err = kafka.CreateTopic(conf.getTopicName(EventBlockGroup))
+	if err != nil {
+		logger.Warn("creating a block group is failed", "err", err)
```
Comment (KimKyungup) [inline datasync/chaindatafetcher/kafka/kafka.go:71]: This routine is for checking if the topic is already created or not?
Why do you need this check routine?

```suggestion
		logger.Warn("already created a block group is failed", "err", err)
```

**Review by ehnuje — DISMISSED** [review]

**Review by KimKyungup — DISMISSED** [review]

**Review by yoomee1313 — DISMISSED** [review]

File: datasync/chaindatafetcher/kafka/kafka.go [hunk 3]
```diff
@@ -60,11 +62,51 @@ func NewKafka(conf *KafkaConfig) (*Kafka, error) {
...
 		admin:    admin,
-	}, nil
+	}
+
+	blockGroupTopic := conf.getTopicName(EventBlockGroup)
+	if err := kafka.setupTopic(blockGroupTopic); err != nil {
+		return nil, err
+	}
+
+	traceGroupTopic := conf.getTopicName(EventTraceBroup)
```
Comment (winnie-byun) [inline datasync/chaindatafetcher/kafka/kafka.go:76]:
```suggestion
	traceGroupTopic := conf.getTopicName(EventTraceGroup)
```
Is this typo?

Comment (jeongkyun-oh) [reply datasync/chaindatafetcher/kafka/kafka.go:76]: Applied. Thanks!

**Review by KimKyungup — DISMISSED** [review]

**Review by ehnuje — DISMISSED** [review]

**Review by aidan-kwon — APPROVED** [review]

**Review by KimKyungup — APPROVED** [review]

**Review by winnie-byun — APPROVED** [review]

**Review by ehnuje — APPROVED** [review]

**Review by yoomee1313 — APPROVED** [review]

**Pull request merged**